types/database/schema/ChannelList.types.ts

```diff
@@ -16,6 +17,11 @@
   isBanned: boolean;
   isMe: boolean;
   image?: string;
+  anon_user_info_emoji_name: string | null;
+  anon_user_info_emoji_code: string | null;
+  anon_user_info_color_name: string | null;
+  anon_user_info_color_code: string | null;
+  isAnonymous: SQLiteBoolean | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding new fields to the `UserSchema` is fine, but make sure that these fields are also handled in all places where this schema is used, such as CRUD operations and any other database interactions. If not, it could lead to inconsistencies or errors in your application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->